Detectors/AOD/src/AODProducerWorkflowSpec.cxx

```diff
@@ -786,6 +677,9 @@ void AODProducerWorkflowDPL::run(ProcessingContext& pc)
           vCollRefsMFT[trackIndex.getIndex()] = collisionID;
         } else if (src == GIndex::Source::ITSTPC) {
           vCollRefsTPCITS[trackIndex.getIndex()] = collisionID;
```

Comment:
The same track may appear in the references of different vertices (if it is not a contributor but is compatible in time). 
I don't see how this is accounted: here you will account the last matching vertex (collisionID) which is not even necessarily the best match.